build/pack.ps1

```diff
@@ -79,12 +79,40 @@ function Pack-Image() {
     }
 }
 
+function Pack-Exe() {
+    param(
+        [string] $Project,
+        [string] $Runtime,
+        [string] $Configuration = $Env:BUILD_CONFIGURATION
+    );
+
+    $OutputPath = Join-Path $Env:BLOBS_OUTDIR $Runtime;
+
+    # Suppress generating pdb files.
+    # See https://github.com/dotnet/cli/issues/2246#issuecomment-320633639.
+    dotnet publish `
+    (Join-Path $PSScriptRoot $Project) `
+        -c $Configuration `
```

Comment:
I'm pretty sure you're missing passing the assembly version.
I think you will also need to pass assembly constants for signed/telemetry.

Reply:
these are all the flags we normally use:
```
        -v $Env:BUILD_VERBOSITY `
        /property:DefineConstants=$Env:ASSEMBLY_CONSTANTS `
        /property:Version=$Env:ASSEMBLY_VERSION `
```

Reply:
Thanks for the catch, somehow I had thought those propagated from the build to the publish step, but that would explain it. I'll fix that now, thanks!